docs/src/app/components/pages/components/TimePicker/Page.js

```diff
@@ -33,6 +36,13 @@ const TimePickersPage = () => (
       <TimePickerExampleSimple />
     </CodeExample>
     <CodeExample
+      title="Inline TimePicker examples"
+      description={descriptions.inline}
```

Comment:
@oliviertassinari : what do you mean? Yes, it picks up the description, as you can see in the screengrab above.